packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -330,7 +330,7 @@ class SystemRulesTable extends React.Component {
                     { remediationsEnabled &&
                         <DataToolbarItem>
                             <ComplianceRemediationButton
-                                allSystems={ [{ id: system.id, ruleObjectsFailed: [] }] }
+                                allSystems={ [{ id: system.id, profiles: system.profiles, ruleObjectsFailed: [] }] }
```

Comment:
:warning:  And here's the other key of the PR - we need to pass the profiles to the ComplianceRemediationButton to find it and pass it